app/src/main/java/bg/zahov/app/data/local/RealmManager.kt

```diff
@@ -69,13 +87,14 @@ class RealmManager {
                 } catch (e: IllegalArgumentException) {
                     throw e
                 }
-                realmInstance!!
-            } else {
-                realmInstance!!
             }
+            realmInstance!!
```

Comment:
In general avoid **!!,** use it only in tests.

Reply:
fix it